### PR TITLE
Downloads are directly accessed from the site

### PR DESCRIPTION
**Pull request by finn00dev**

### CODE REVIEW

File: www/html/ci/application/views/landing.php [hunk 1]
```diff
@@ -20,12 +20,12 @@ function googleTranslateElementInit() {
 	}
 	</script><script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
 	<script>
-	
 
-	
-	
+
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php:24]: @finn00dev  Why am I seeing blank lines here being changed?

File: www/html/ci/application/views/landing.php [hunk 2]
```diff
@@ -578,7 +578,7 @@ function googleTranslateElementInit() {
 <p>Do you want to play an arcade game that has you flying a jetplane while playing epic music in space? Avoid the space obstacles and enemies and get to the end of the song! If you get hit, the space gods will give you another chance by having you press a keyboard key within a certain time limit. If you press it in time, then you are safe! If not, then I guess you are starting the song over again. Inspired by my childhood space arcade games and The Impossible Game.</p>
 
 <p><strong>Watch the trailer </strong><a href="https://youtu.be/tYUymiCDOY8" target="_blank">here!</a></p>
-<p><strong>Download the game! </strong><a href="http://www.mediafire.com/file/7fdkazh3w61l6dc/SpaceDodgersGithub.zip" target="_blank">WindowsOS</a></p>
+<p><strong>Download the game! </strong><a href="downloads/SpaceDodgersGithub.zip" download="SpaceDodgers">WindowsOS</a></p>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: @finn00dev Is the name of the downloaded file "SpaceDodgers" or "SpaceDodgersGithub.zip"? Let's not change the downloaded filename.

File: www/html/ci/application/views/landing.php [hunk 3]
```diff
@@ -669,9 +669,9 @@ function googleTranslateElementInit() {
 <p>Combine the cuteness of mochi sticky rice balls and a classic game of speed hockey. You get Dango Puck! With the exception of the music, everything was made by me. All graphics, code, and audio clips were made by me.</p>
 
 <p><strong>Download the game for </strong>
-<a href="http://www.mediafire.com/file/ty75vbtmqx5450m/DangoPuckv1MacUniversal.zip" target="_blank" >MacOS</a>, 
-<a href="http://www.mediafire.com/file/yl1w0b261212heb/DangoPuckv1Win86.zip" target="_blank" >Windows</a>, or
-<a href="http://www.mediafire.com/file/hpelalbkzni12m2/DangoPuckv1LinuxUniversal.zip" target="_blank" >Linux Universal.</a>
+<a href="downloads/DangoPuckv1MacUniversal.zip" download="DangoPuckv1Mac" >MacOS</a>,
+<a href="downloads/DangoPuckv1Win86.zip" download="DangoPuckv1Win86" >Windows</a>, or
+<a href="downloads/DangoPuckv1LinuxUniversal.zip" download="DangoPuckv1Linux" >Linux Universal.</a>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: @finn00dev Let's keep the filename the same.

**Review by ktorii — COMMENTED** [review]
@finn00dev These downloads should not be in the views folder. They are not views. Move them to where the js, css files are. I think it's in www/html/

**Comment by finn00dev** [comment]
Hey Ken,
just in response to your comments, I put the files in the views folder just to make it easier for the tag to access them but I understand now that was the wrong place, will put them in the html folder.

**Comment by finn00dev** [comment]
Ill also fix the other issues

**Pull request closed without merging**